adapters/handlers/rest/configure_api.go

```diff
@@ -148,6 +148,10 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 		err := sentry.Init(sentry.ClientOptions{
 			Dsn:              appState.ServerConfig.Config.Sentry.DSN,
 			Debug:            appState.ServerConfig.Config.Sentry.Debug,
+			Release:          "weaviate@" + config.DockerImageTag,
```

Comment:
just a suggestion, isn't it better to rename it to just `DockerImageTag` also? `Release` is misleading I think..

Reply:
@antas-marcin this `Release` is from the sentry option struct and that's the naming it expects. Can't change it sadly. Functionaly it's going to be identical to a releasy anyway (a docker image tag points to a release in our flow)